### PR TITLE
Add TO Go monitoring route prepared statements

### PR DESCRIPTION
**Pull request by rob05c**

### CODE REVIEW

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/incubator-trafficcontrol-PR-trafficops-test/60/

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/incubator-trafficcontrol-traffic_ops-test-PR/62/

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/incubator-trafficcontrol-PR/100/

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/incubator-trafficcontrol-traffic_ops-test-PR/64/

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/incubator-trafficcontrol-PR-trafficops-test/62/

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/incubator-trafficcontrol-PR/102/

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/incubator-trafficcontrol-PR-trafficops-test/63/

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/incubator-trafficcontrol-traffic_ops-test-PR/65/

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/incubator-trafficcontrol-PR/103/

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/incubator-trafficcontrol-PR/140/

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/incubator-trafficcontrol-PR-trafficops-test/101/

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/incubator-trafficcontrol-whatchanged/6/

File: traffic_ops/traffic_ops_golang/routes.go [hunk 1]
```diff
@@ -34,29 +34,30 @@ func Routes(d ServerData) ([]Route, http.Handler, error) {
...
-		{1.2, http.MethodGet, "cdns/{cdn}/configs/monitoring", wrapHeaders(wrapAuth(monitoringHandler(d.DB), d.Insecure, d.TOSecret, rd.PrivLevelStmt, MonitoringPrivLevel))},
-		{1.2, http.MethodGet, "cdns/{cdn}/configs/monitoring.json", wrapHeaders(wrapAuth(monitoringHandler(d.DB), d.Insecure, d.TOSecret, rd.PrivLevelStmt, MonitoringPrivLevel))},
+		{1.2, http.MethodGet, "cdns/{cdn}/configs/monitoring", wrapHeaders(wrapAuth(monitoringHandler(rd.Monitoring), d.Insecure, d.TOSecret, rd.PrivLevelStmt, MonitoringPrivLevel))},
+		{1.2, http.MethodGet, "cdns/{cdn}/configs/monitoring.json", wrapHeaders(wrapAuth(monitoringHandler(rd.Monitoring), d.Insecure, d.TOSecret, rd.PrivLevelStmt, MonitoringPrivLevel))},
 	}, rootHandler(d), nil
 }
 
 type RouteData struct {
 	PrivLevelStmt *sql.Stmt
+	Monitoring    *MonitoringData
```
Comment (dewrich) [inline traffic_ops/traffic_ops_golang/routes.go]: I think this is to specific at this level.  So, the developer will have to implement this for every single route?

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/incubator-trafficcontrol-PR/235/

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/incubator-trafficcontrol-PR-trafficops-test/193/

**Comment by rob05c** [comment]
Closing this, as I did performance testing and am not seeing a significant speed improvement. Which surprises me, but I don't have time to investigate further.

**Pull request closed without merging**